.gitignore

```diff
@@ -1 +1,3 @@
 node_modules/
+*npm-debug*
+*yarn-error*
```

Comment:
What do the '*'s do? Mind @lumodon in your response so I can learn, or echoing me? Just curious.